framework_agreement_sourcing/model/logistic_requisition.py

```diff
@@ -38,37 +36,26 @@ def _prepare_line_source(self, cr, uid, line,
         rest as 'other'. Those are default value that can be changed afterward
         by the user.
 
-        :params line: browse record of origin requistion.line
-        :params agreement: browse record of origin agreement
-        :params qty: quantity to be set on source line
+        :param self: record set of origin requistion.line
```

Comment:
A detail, but I wouldn't documents things that are not specific to this method like `self`.